modules/nw-metallb-configure-address-pool.adoc

```diff
@@ -0,0 +1,53 @@
+[id="nw-metallb-configure-address-pool_{context}"]
+= Configuring an address pool
+
+Add a cluster administrator, you can add address pools to your cluster to control the IP addresses that MetaLLB can assign to load-balancer services.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
```

Comment:
metallb-system namespace must be already created. Therefore MetalLB operator needs to installed before address pool is configured.

Reply:
Adding the namespace is covered during the installation of the operator.